ddtrace/contrib/django/__init__.py

```diff
@@ -12,6 +12,13 @@
         'ddtrace.contrib.django',
     ]
 
+Add the Datadog middleware to your MIDDLEWARE_CLASSES:
+
+  MIDDLEWARE_CLASSES = [
+      'ddtrace.contrib.django.TraceMiddleware',
```

Comment:
With new version it's not required anymore. Adding the Django app is enough (or did you spot a bug?)

Reply:
@palazzem a user reached out on 07 nov with this issue.